src/java.base/share/classes/java/text/DecimalFormatSymbols.java

```diff
@@ -88,7 +90,7 @@ public class DecimalFormatSymbols implements Cloneable, Serializable {
      * @see java.util.Locale.Category#FORMAT
      */
     public DecimalFormatSymbols() {
-        initialize( Locale.getDefault(Locale.Category.FORMAT) );
+        initialize(Locale.getDefault(Locale.Category.FORMAT));
```

Comment:
Taking this opportunity, remove the surrounding spaces in the method itself: (L:817)